pkg/operator/staticpod/controller/installer/installer_controller.go

```diff
@@ -90,9 +92,12 @@ type InstallerController struct {
 
 	installerPodMutationFns []InstallerPodMutationFunc
 
+	startupMonitorDeployed func() (bool, error)
+
 	factory          *factory.Factory
 	clock            clock.Clock
 	installerBackOff func(count int) time.Duration
+	fallbackBackOff  func(count int) time.Duration
```

Comment:
have you forgotten to wire `fallbackBackOff`?

Reply:
lost in rebase.

Reply:
fixed